roles/ipaclient/vars/Debian-buster.yml

```diff
@@ -0,0 +1,3 @@
+# vars/Debian.yml
+ipaclient_packages: [ "freeipa-client" ]
+ansible_python_interpreter: '/usr/bin/python2'
```

Comment:
Is this really needed for all Debian (based) releases?

Reply:
At least it seems to be for Debian buster, which this file is named for for the `with_first_found` conditional include statement.